src/filing/actions/receiveEdit.js

```diff
@@ -1,14 +1,66 @@
+import React from 'react'
 import * as types from '../constants'
 
+const PATTERN_1111 = /1111/
```

Comment:
do we need to worry about number formatting for this, or is it always going to be just 4 `1`s in a row? i.e. `1,111`, `1,111.00`, etc; also this pattern would capture anything with `1111` in it as well, is that desired? i.e. `121111324 a street`, if we only care about the sequence of 4 `1`s, we could do `/^1111$/

Reply:
From my testing we are only concerned about `1111`.  When entering `1111` in a Float field, it gets converted to `1111.0` which is covered by our current pattern.  With that in mind, `/^1111$/` seems too restrictive but `/^1111/` is probably still too loose of a match.  
 
 I think `/^1111(\.0)?/` covers the instances I've seen but I will double check with the backend folks to identify other edge cases. 👍🏽